pod/perlfunc.pod

```diff
@@ -9608,18 +9608,20 @@ L<C<use>|/use Module VERSION LIST>ing library modules that won't work
 with older versions of Perl.
 (We try not to do this more than we have to.)
 
-C<use VERSION> also lexically enables all features available in the requested
+C<use VERSION> lexically enables all features available in the requested
 version as defined by the L<feature> pragma, disabling any features
 not in the requested version's feature bundle.  See L<feature>.
-Similarly, if the specified Perl version is greater than or equal to
+If the specified Perl version is greater than or equal to
 5.12.0, strictures are enabled lexically as
-with L<C<use strict>|strict>.  Any explicit use of
+with L<C<use strict>|strict>.  
+Similarly, L<warnings> are enabled if C<VERSION> is 5.35.0 or higher.
+Any explicit use of
 C<use strict> or C<no strict> overrides C<use VERSION>, even if it comes
-before it.  Later use of C<use VERSION>
-will override all behavior of a previous
-C<use VERSION>, possibly removing the C<strict> and C<feature> added by
-C<use VERSION>.  C<use VERSION> does not
-load the F<feature.pm> or F<strict.pm>
+before it (likewise for C<use warnings> or C<no warnings>).
```

Comment:
This isn't true, a `no warnings` before a `use 5.36` will enable warnings.

Reply:
Thanks.  I removed that statement.